tracer/build/_build/Build.Steps.Debugger.cs

```diff
@@ -62,8 +62,8 @@ partial class Build
         .Requires(() => DebugType != null)
         .Executes(() =>
         {
-            DotnetBuild(DebuggerUnreferencedExternal, framework: Framework, noDependencies: false);
-            DotnetBuild(DebuggerSamplesTestRuns, framework: Framework, noDependencies: false);
```

Comment:
We don't want to build for _all_ frameworks here, as we parallelize each of these in CI

Reply:
reverted unrelated clean up :)